File: app/src/main/java/org/ametro/ui/controllers/MotionEventMultiTouchWrapper.java
package org.ametro.ui.controllers;

import android.view.MotionEvent;

public class MotionEventMultiTouchWrapper extends MotionEventWrapper {

	public MotionEventMultiTouchWrapper(MotionEvent event) {
		super(event);
	}

	public int getAction(){
		return event.getAction() & MotionEvent.ACTION_MASK;
	}
	
	public float getX(int pos){
		return event.getX(pos); 
	}
	
	public float getY(int pos){
		return event.getY(pos);
	}
		
}

File: app/src/main/java/io/github/romangolovanov/apps/ametro/ui/controllers/MotionEventMultiTouchWrapper.kt
package io.github.romangolovanov.apps.ametro.ui.controllers

import android.view.MotionEvent

class MotionEventMultiTouchWrapper(event: MotionEvent) : MotionEventWrapper(event) {

    override fun getAction(): Int = event.action and MotionEvent.ACTION_MASK

    override fun getX(pos: Int): Float = event.getX(pos)

    override fun getY(pos: Int): Float = event.getY(pos)
}
